lib/hyrax/controlled_vocabularies/location.rb

```diff
@@ -56,6 +59,27 @@ def rdf_label
       # rubocop:enable Metrics/CyclomaticComplexity
       # rubocop:enable Metrics/PerceivedComplexity
 
+      # ActiveTriples just grabs the first label
+      # This is typically fine because it will find the preferred label first
+      # But sometimes the first preferred label isn't in english, or all aren't
+      # Disable rubocop because just barely fails abcsize
+      # rubocop:disable Metrics/AbcSize
+      def rdf_ons_label
+        labels = Array.wrap(self.class.rdf_label)
+        labels += default_labels
+        # OVERRIDE from rdf_triples to select only and all english labels
+        values = []
+        labels.each do |label|
+          values += get_values(label).to_a
```

Comment:
I guess += does the same thing as << for arrays? I can see you are making a multidimensional array here but I would have expected to see << for pushing things into an array instead of +=

Reply:
I tried this and failed test. I think `#get_values` is returning an array, so array addition makes sense here